ts/src/binance.ts

```diff
@@ -3677,15 +3677,16 @@ export default class binance extends Exchange {
         if (!market['spot']) {
             throw new NotSupported (this.id + ' editSpotOrder() does not support ' + market['type'] + ' orders');
         }
-        const request = {
-            'symbol': market['id'],
-            'side': side.toUpperCase (),
-        };
         const clientOrderId = this.safeStringN (params, [ 'newClientOrderId', 'clientOrderId', 'origClientOrderId' ]);
         let response = undefined;
         if (market['spot']) {
-            response = await this.privatePostOrderCancelReplace (this.extend (request, params));
+            const payload = this.editSpotOrderRequest (id, symbol, type, side, amount, price, params);
+            response = await this.privatePostOrderCancelReplace (payload);
         } else {
+            const request = {
```

Comment:
@ttodua something is wrong here, we're inside `editSpotOrder` but we have an else branch that handles swap orders?

Reply:
yeah, i haven't looked around, i was looking just spot order request corrections... something was off yeh. 
also, maybe we neither need `if (!market['spot'])` check inside `editSpotOrder`.

Reply:
@ttodua yeah I removed all of that

Reply:
it might need later to be checked inside `editSwapOrder` or alike, i've not looked in other areas yet.